src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/impl/GitClientImpl.java

```diff
@@ -195,13 +195,19 @@ private boolean pull() {
 		try {
 			List<RemoteConfig> remotes = git.remoteList().call();
 			for (RemoteConfig remote : remotes) {
-				git.fetch().setRemote(remote.getName()).setRefSpecs(remote.getFetchRefSpecs()).call();
+				git.fetch()
+					.setRemote(remote.getName())
+					.setRefSpecs(remote.getFetchRefSpecs())
+					.setRemoveDeletedRefs(true)
+					.call();
+				LOGGER.info("Fetched branches in "+git.getRepository().getDirectory());
```

Comment:
Hi @lw2011, 

thanks for the changes, they look good! The only thing I would like to ask you: Please slightly change the formatting and put spaces in front and after operators (+).

Please merge the branch branch afterwards. Thanks!

Cheers,
Anja